source/stylesheets/refills/_device.scss

```diff
@@ -45,16 +30,18 @@
 
     h4 {
       border-bottom: 1px solid transparentize($device-text-color, 0.7);
-      margin-bottom: 0.5em;
+      font-size: 1.5em;
+      margin: 0 0 0.5em 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 0.5em` instead of `0 0 0.5em 0`